### PR TITLE
drawers spawn with underwear + eye fix

### PR DESCRIPTION
**Pull request by NullFag**

### CODE REVIEW

File: code/game/objects/structures/dresser.dm [hunk 1]
```diff
@@ -19,6 +19,36 @@
...
+	STR.max_items = 40
+
+	if(mapload)
+		generate_underwear()
+
+#define UNDIE_AMOUNT 11
+/obj/structure/dresser/proc/generate_underwear()
+	var/static/list/possible_underwear = subtypesof(/obj/item/clothing/underwear/briefs)
+	var/static/list/possible_undershirt = subtypesof(/obj/item/clothing/underwear/shirt)
+	var/static/list/possible_socks = subtypesof(/obj/item/clothing/underwear/socks)
```
Comment (SandPoot) [inline code/game/objects/structures/dresser.dm:36]: `typesof()` root path is still valid

Comment (NullFag) [reply code/game/objects/structures/dresser.dm:36]: Wasn't sure so I used subtypes

Comment (SandPoot) [reply code/game/objects/structures/dresser.dm:36]: Happy to help

Comment (SandPoot) [reply code/game/objects/structures/dresser.dm:36]: also, i'd recommend storing it in a global list, or at least a static list outside of the proc

**Pull request closed without merging**